File: samples/JavaScript/Json Store/Cursor/Select/By Partial Key.js
session.setAsync(false);

var store = session.getDatabase("playground").getStore("pinball");
var s = "";

s += ">>> Select a document by Partial Key\n";
var c = store.openCursor().range(0,5);

// Select by Partial Key
c.partialKey("11").find(function(e) {
  s += "  Key:"+e.getKey()+"\n";
});  
s += "\n";


darwino.Utils.setText("content","{0}",s);

File: samples/Java/Json Store/Cursor/Select/By Partial Key.java
Store store = session.getDatabase("playground").getStore("pinball");
String s = ""; final StringBuilder b = new StringBuilder();

s += ">>> Select a document by partial key\n";
Cursor c = store.openCursor().range(0,5);
b.setLength(0);
c.partialKey("11").find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  Key:"+e.getKey()+"\n");
    return true;
  }
});  
s += b.toString()+"\n";


_formatText("{0}",s);